pkg/cmd/stack/translate.go

```diff
@@ -116,7 +116,8 @@ func translateConfigMap(s *model.Stack) *apiv1.ConfigMap {
 func translateDeployment(svcName string, s *model.Stack) *appsv1.Deployment {
 	svc := s.Services[svcName]
 
-	healthcheckProbe := getSvcProbe(svc)
+	svcHealthchecks := getSvcProbe(svc)
```

Comment:
maybe we should either change function name or change variable name? I would go to rename function name to `getSvcHealthProbes`